packages/sdk/src/index.test.ts

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Nice, thanks for creating a mocked test 💪

Would love to backport this to the `@epilot/auth` package.

I'm not really in love with the idea of getting rid of the monorepo and implementing the whole sdk in one massive package.